### DISCUSSION TITLE
SMTP and email

### DISCUSSION THREAD

**Discussion by doudousupreme** [jef/streetmerchant#1564] [Help]
Hello,

I have setup streetmerchant, but I can't manage to get the email notification working.
I wanted to use an outlook email, and tried to set the SMTP field but it fails to connect:
"couldn't send email connect ETIMEDOUT"

EMAIL_PASSWORD="MY PASSWORD HERE"
EMAIL_USERNAME="MY EMAIL HERE"
SMTP_ADDRESS="smtp-mail.outlook.com"
SMTP_PORT="3000"
IN_STOCK_WAIT_TIME="90"
SHOW_ONLY_SERIES="3080,rx6800xt"
MAX_PRICE_SERIES_3080="1300"
MAX_PRICE_SERIES_RX6800XT="1300"
STORES="bestbuy-ca,canadacomputers,amazon-ca,amd-ca,memoryexpress,newegg-ca,walmart-ca,nvidia"
COUNTRY="canada"

I have tried different ports just in case. I also tried to ping smtp-mail.outlook.com then replace it with the IP address but got the same issue. Could someone help me out with the settings required to get the email to work?

**Comment by doudousupreme** [comment]
Ok it's **RESOLVED**. I manage to get it to work with the following settings. I actually notice after that two of my attempts worked once Outlook told me to approve the spam I was sending to myself :)

SMTP_ADDRESS="smtp-mail.outlook.com"
SMTP_PORT="587"

Make sure to check your emails because they could send back an error if you don't approve like me.
"To continue sending messages, please sign in and validate your Outlook.com account.
This helps us stop automated programs from sending junk email.
Thanks for your help and patience!"

**Comment by lec6213** [comment]
Yup, your outgoing port depending on protocol can often be different.  Also, anyone sending via Gmail, Gmail has a setting called "Les Secured Apps."  If you are getting rejections through Gmail, google that setting, login and enable the use of less secure, and then Gmail will permit SMTP usage via STREETMERCHANT.  This setting can also randomly disengage as well, so you may have to parodically go in and re-engage it too, at least I have had too.
